tmt/steps/provision/podman.py

```diff
@@ -138,7 +135,7 @@ def start(self):
 
         # Run the container
         self.debug(f"Start container '{self.image}'.")
-        self.container_id = self.podman(
+        self.podman(
             ['run'] + workaround +
             ['--name', self.container, '-v', f'{workdir}:{workdir}:z',
              '-itd', self.image])[0].strip()
```

Comment:
We could remove also `[0].strip()` as returned value is ignored

Reply:
Right, removed.